pkg/kconfig/kconfig.go

```diff
@@ -25,13 +25,19 @@ const (
 	containerCgroupPath = "/proc/1/cgroup"
 )
 
-var ebpfOptions = []string{
-	"CONFIG_BPF",
-	"CONFIG_BPF_SYSCALL",
-	"CONFIG_HAVE_EBPF_JIT",
-	"CONFIG_BPF_JIT",
-	"CONFIG_BPF_JIT_DEFAULT_ON",
-	"CONFIG_BPF_EVENTS",
+type ebpfOption struct {
+	name string
+	// Used for specifying synonymous kernel options
+	alternatives []string
+}
+
+var ebpfOptions = []ebpfOption{
+	{name: "CONFIG_BPF"},
+	{name: "CONFIG_BPF_SYSCALL"},
+	{name: "CONFIG_HAVE_EBPF_JIT"},
+	{name: "CONFIG_BPF_JIT"},
+	{name: "CONFIG_BPF_JIT_ALWAYS_ON", alternatives: []string{"CONFIG_BPF_JIT_DEFAULT_ON"}},
```

Comment:
Actually, we should just have ARCH_WANT_DEFAULT_BPF_JIT instead of CONFIG_BPF_JIT_DEFAULT_ON as an alternative. That will handle configs in other archs and older kernels too. So, while you're at it, can you please change this? Thanks.

Reply:
Done!